preset-io/incubator-superset
# Sort select inputs in Filter Config modal alphabetically

### SUMMARY
The aim of this PR is to sort select inputs (datasource & field inputs located in Filter Config Modal) alphabetically. 

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
**before:**
![image](https://user-images.githubusercontent.com/47450693/101058620-9ddb7c00-358d-11eb-84be-87935c393a49.png)
![image](https://user-images.githubusercontent.com/47450693/101058627-9fa53f80-358d-11eb-8d69-eb50090e6fa0.png)

**after:**
![image](https://user-images.githubusercontent.com/47450693/101058639-a2079980-358d-11eb-8ab7-2572ea307721.png)
![image](https://user-images.githubusercontent.com/47450693/101058650-a469f380-358d-11eb-945b-90360fc68a7b.png)

### TEST PLAN
Verify manually:
1. Open Filter Configuration modal
2. Add new filter (or edit existing one)
3. Choose datasource and then field.

### ADDITIONAL INFORMATION



cc @junlin @villebro
